### ISSUE TITLE
Core: At most N instances of a mandate running at a time

### ISSUE THREAD

**Issue by justinp** [scalawag/jibe#15]
Right now, all mandates are executed serially within a Commander and in parallel across Commanders.  It would be nice to have a way to make it so that a particular mandate is limited in the number of Commanders it can be run on simultaneously.  This would be useful for operations such as rolling restarts.

Ideally, this should be a Mandate wrapper that is itself a Mandate, so that it can be used with any Mandate, including composites.
